types/emoji-mart/dist-es/utils/shared-props.d.ts

```diff
@@ -92,4 +92,7 @@ export interface PickerProps {
     notFoundEmoji?: string;
     icons?: CustomIcons;
     darkMode?: boolean;
+    theme?: 'auto' | 'light' | 'dark';
```

Comment:
These parameters are only available on 3.0.X. I'd suggest updating the version accordingly (in index.d.ts)

Example: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43908 (PR introducing the theme prop and changing version to 3.0)